include/eve/module/math/constant/two_o_pi.hpp

```diff
@@ -0,0 +1,78 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/module/core.hpp>
+
+namespace eve
+{
+  //================================================================================================
+  //! @addtogroup core
+  //! @{
+  //! @var two_o_pi
+  //!
+  //! @brief Callable object computing the two_o_pi constant value : \f$2/\pi\f$.
```

Comment:
Same. This is not a constant with a name so just write

computing the \f$\frac{2}{\pi}\f$ constant value.